client/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,54 @@
+.blockquote {
+  @include font('HNM2');
+  text-align: center;
+
+  &--wb {
+    @include font('WB5');
+  }
+
+  &--lr {
+    @include font('LR2');
+    text-align: left;
+  }
+
+  footer {
```

Comment:
In general, I'd say definitely 'yes'. If we readily have the capacity to add classes to all the HTML we receive, then we probably should.

This is styled with the assumption that we're only going to be able to class the `blockquote` element itself (and scopes all other styling to that element). I'd say this approach is acceptable for html structures which are fairly well-defined, such as `blockquote`s and `table`s. If we don't have to here, then I'd agree that we probably shouldn't.

Will the parsing we do be able to add classes to each element?